libtransmission/session-settings.h

```diff
@@ -49,6 +49,7 @@ struct tr_variant;
     V(TR_KEY_port_forwarding_enabled, port_forwarding_enabled, bool, true, "") \
     V(TR_KEY_preallocation, preallocation_mode, tr_preallocation_mode, TR_PREALLOCATE_SPARSE, "") \
     V(TR_KEY_prefetch_enabled, is_prefetch_enabled, bool, true, "") \
+    V(TR_KEY_proxy_url, proxy_url, std::string, "", "") \
```

Comment:
This new setting should be documented in 

`docs/Editing-Configuration-Files.md`

Reply:
Add documentation to docs/Editing-Configuration-Files.md
Should I remove old documentation to deprecated proxy-settings?

Reply:
Hmmm good question, thanks for asking.

Probably better to add the documentation as long as it's clearly marked as deprecated.

Reply:
Old settings already marked as deprecated. I don't changed it.